README.md

```diff
@@ -16,13 +16,11 @@ Labels are used for helping to filter bugs into groups but also to track the sta
 - `status-contactready` - Issue has been analyzed and is ready for someone to contact the site
 - `status-duplicate` - Issue is the same as an already-reported issue
 - `status-fixed` - Issue is fixed
-- `status-ie-bugbar` - The bug is listed on internet explora
-- `status-imported` - The issue is cloned from another site
 - `status-invalid` - Issue is not a web compatibility issue
-- `status-needscontact` - The web site with the issue needs to be contancted
+- `status-needscontact` - The web site with the issue needs to be contacted
 - `status-needsdiagnosis` - Issue needs further analysis to find the cause
 - `status-needsinfo` - Issue needs more information
-- `status-needstriage` - The issue needs to be screened and priorized
+- `status-needstriage` - The issues needs to be screened and  prioritized
```

Comment:
@techchic Thanks so much, just one minor thing. Issues and needs are both plural. "The issue needs" is fine. :)